redis/endpoint.bal

```diff
@@ -16,26 +16,32 @@
 
 import ballerina/jballerina.java;
 
-# Represents Redis client endpoint.
+# Ballerina Redis connector provides the capability to access Redis cache.
+# This connector lets you to perform operations to access and manipulate key-value data stored in a Redis database. 
+# 
 @display {label: "Redis Client", iconPath: "RedisLogo.png"}
-public client class Client {
+public isolated client class Client {
     private ClientEndpointConfiguration clientEndpointConfig = {};
 
-    handle datasource;
+    final handle datasource;
 
-    # Gets called when the endpoint is being initialized during the module initialization.
-    public function init(ClientEndpointConfiguration config) returns error? {
+    # Initializes the connector. During initialization you have to pass needed configurations such as host name, port number and password.
+    #
+    # + config - Configuration for the connector
+    # + return - `http:Error` in case of failure to initialize or `null` if successfully initialized
```

Comment:
Suggestion:
`http:Error` in case of failure to initialize, or `null` if successfully initialized